### PR TITLE
Add support for v7.x CER Component SDK

### PR DESCRIPTION
**Pull request by sm6srw**
### Purpose

This pull request add support for the latest CER tool.

### Declarations

Check these if you believe they are true

- [X] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [x] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [X] This PR contains no files larger than 50 MB

### Release Notes

Add support for v7.x CER Component SDK

### Reviewers

(FILL ME IN) Reviewer 1 (If possible, assign the Reviewer for the PR)

(FILL ME IN, optional) Any additional notes to reviewers or testers.

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

File: src/DynamoCoreWpf/Utilities/CrashReportTool.cs [hunk 1]
```diff
@@ -233,30 +233,43 @@ internal static bool ShowCrashErrorReportWindow(DynamoViewModel viewModel, Crash
...
+                        cerDLL.RegisterUPI(upiConfigFilePath);
+
+                        if (!string.IsNullOrEmpty(dynName))
+                        {
+                            cerDLL.SetStringParam(ReportStringParamKey.StringKeyDwg, dynName);
+                        }
+
+                        foreach (var file in filesToSend)
+                        {
+                            cerDLL.SetMultiStringParam(ReportMultiStringParamKey.MultiStringKeyExtraFile, file, filesToSend.Count + 1);
```
Comment (pinzart90) [inline src/DynamoCoreWpf/Utilities/CrashReportTool.cs:262]: will `SetMultiStringParam` actually send the file content ?

Comment (pinzart90) [reply src/DynamoCoreWpf/Utilities/CrashReportTool.cs:262]: why filesToSend.Count + 1 ?

**Comment by pinzart90** [comment]
@sm6srw  did you test this out ? There is a debug mode to cause crashes on node placement (if I remember correctly)

**Comment by sm6srw** [comment]
Yes, I have tested it. I have checked the various logs and the result on the CER site.

File: src/DynamoCoreWpf/Utilities/CrashReportTool.cs (hunk 1 above)
Comment (sm6srw) [reply src/DynamoCoreWpf/Utilities/CrashReportTool.cs:262]: I can maybe remove +1. Let me check.

File: src/DynamoCoreWpf/Utilities/CrashReportTool.cs [hunk 2]
```diff
@@ -187,78 +187,75 @@ internal static bool ShowCrashErrorReportWindow(DynamoViewModel viewModel, Crash
...
                     {
-                        dynConfig = $"/DWG {dynName}";
+                        cerDLL.SetStringParam(ReportStringParamKey.StringKeyDwg, dynName);
                     }
 
-                    var miniDumpFilePath = CreateMiniDumpFile(cerDir.FullName);
-                    var upiConfigFilePath = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "upiconfig.xml");
+                    foreach (var file in filesToSend)
+                    {
+                        cerDLL.SetMultiStringParam(ReportMultiStringParamKey.MultiStringKeyExtraFile, file, filesToSend.Count);
```
Comment (pinzart90) [inline src/DynamoCoreWpf/Utilities/CrashReportTool.cs:247]: Does this function call actually send out the files or just the file paths ?

Comment (pinzart90) [reply src/DynamoCoreWpf/Utilities/CrashReportTool.cs:247]: OK..I saw the file content on some of the reports

**Review by pinzart90 — APPROVED** [review]

**Pull request merged**